client/src/admin/adminPanel.tsx

```diff
@@ -156,7 +157,10 @@ const AdminPanel = ({
           <AndroidRow key={'rpi-' + i}>
             <AndroidHeader>{device.name}</AndroidHeader>
             <LocationRow>
-              N {device.lat.toFixed(6)}° E {device.lon.toFixed(6)}°
+              N {device.lat.toFixed(6)}° E {device.lon.toFixed(6)}° x:
+              {geoCoordsToPlaneCoords(device, device.height).x} y:
+              {geoCoordsToPlaneCoords(device, device.height).y} z:
+              {device.height}
```

Comment:
I think that using = instead of : would improve readability, i.e. x=... y=... z=... Also, consider rounding to 6 decimal places here, as well.